Source/FlowEditor/Private/Graph/FlowGraphEditor.cpp

```diff
@@ -561,25 +552,53 @@ bool SFlowGraphEditor::CanCutNodes() const
 
 void SFlowGraphEditor::CopySelectedNodes() const
 {
-	const FGraphPanelSelectionSet SelectedNodes = GetSelectedNodes();
-	for (FGraphPanelSelectionSet::TConstIterator SelectedIt(SelectedNodes); SelectedIt; ++SelectedIt)
+	// Export the selected nodes and place the text on the clipboard
```

Comment:
Copy and Paste underwent a major revision to handle copying and pasting a tree of subnodes.